tests/unit/experiments/experiment_tests.py

```diff
@@ -529,6 +529,88 @@ def test_that_bucketing_events_not_sent_if_cant_find_experiment(self):
         experiments.variant("test", user=self.user)
         self.assertEqual(self.event_logger.log.call_count, 0)
 
+    def test_none_returned_on_variant_call_with_bad_id(self):
+        self.mock_filewatcher.get_data.return_value = {
+            "test": {
```

Comment:
what is bad about the id here?  I can't tell.

Reply:
it's a string rather than an int

Reply:
ah!